pkg/apis/hive/v1/clusterdeployment_types.go

```diff
@@ -128,6 +128,10 @@ type ClusterDeploymentSpec struct {
 	// time since the cluster last came out of hibernation.
 	// +optional
 	HibernateAfter *metav1.Duration `json:"hibernateAfter,omitempty"`
+
+	// LimitInstallRestarts is the number of times Hive attempts to restart an installation. If the last attempt
+	// fails ClusterDeployment will show an error in the status
+	LimitInstallRestarts *int `json:"limitInstallRestarts,omitempty"`
```

Comment:
Make this a noun instead of a verb.
```suggestion
	InstallRestartLimit *int `json:"installRestartLimit,omitempty"`
```